.travis.yml

```diff
@@ -17,50 +17,47 @@
 # along with INSPIRE; if not, write to the Free Software Foundation, Inc.,
 # 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
 
-addons:
-  postgresql: "9.4"
-  apt:
-    packages:
-      - nodejs
-
 notifications:
   email: false
 
+# Not needed at the moment since we use Docker.
+# If this kicks in we can try to use the cache
 sudo: false
 
 language: python
 
-cache:
-  directories:
-    - $HOME/.cache/pip
-    - $HOME/.nvm
-
 services:
-  - postgresql
-  - elasticsearch
-  - rabbitmq
-  - redis
+  - docker
 
 env:
-  - SQLALCHEMY_DATABASE_URI="postgresql+psycopg2://postgres@localhost:5432/inspirehep"
+  global:
+    - DOCKER_DATA="$HOME/docker_data"
+    - DOCKER_VERSION=1.10.1-0~trusty
 
 python:
   - "2.7"
 
 before_install:
-  - "travis_retry pip install --upgrade pip setuptools py mock pytest"
-  - "travis_retry pip install twine wheel coveralls"
-  - "npm update && npm install --silent -g node-sass clean-css uglify-js requirejs"
+  - pip install docker-compose
```

Comment:
What's the reason we don't use the linked post's solution for fixing `docker-compose` version?